ios/FluentUI/Navigation/Views/LargeTitleView.swift

```diff
@@ -70,6 +70,10 @@ class LargeTitleView: UIView {
 
     var onAvatarTapped: (() -> Void)? // called in response to a tap on the MSAvatarView
 
+    var avatarUIView: UIView? {
```

Comment:
this is incorrect. largetitleview has 2 avatar views currently (avatarView, smallMorphingAvatarView). Small one when it is collapsed header, and large one.

Reply:
is only one of these views visible at a time? should this return the currently visible one?

Reply:
Addressed this in the most recent commit, also validated that in both expanded and compact mode the `UIPopoverPresentationController` anchors to the correct view

Reply:
"avatarUIView" doesn't sound very swift friendly name. Also, there might not be any avatar view visible depending on how navigation bar is set up

Reply:
agreed, unfortunately can't use `avatarView` since it collides with the name of the internal property. if both avatars are nil this will return nil as intended. is there a case where avatar data is set but it is not visible?

Reply:
you would have to test it. it isn't the exact scenario but in demo app > navigationcontroller > "show without avatar" > "select" button. You can imagine that there are scenarios where client lets users to do cell selection operation while hiding avatar view. either it is a var with different name(like visiableAvatarView) or i suggest making a function like visibleAvatar()->UIView?  to make it clearer that it will return nil when showsProfileButton is false.

Reply:
great points, will test invisible avatar scenario and push changes

Reply:
i've validated nil is returned when avatar is unset/nil + when cell selection/search bar focus hides the title view